frontend/public/components/notification-drawer.tsx

```diff
@@ -105,25 +108,47 @@ const getAlertNotificationEntries = (
     : [];
 
 const getUpdateNotificationEntries = (
+  cv: ClusterVersionKind,
   isLoaded: boolean,
-  updateData: ClusterUpdate[],
   isEditable: boolean,
   toggleNotificationDrawer: () => void,
-): React.ReactNode[] =>
-  isLoaded && !_.isEmpty(updateData) && isEditable
+): React.ReactNode[] => {
+  const updateData: ClusterUpdate[] = getSortedUpdates(cv);
+  const currentChannel = cv?.spec?.channel;
+  const currentPrefix = splitClusterVersionChannel(currentChannel)?.prefix;
+  const similarChannels = getSimilarClusterVersionChannels(currentPrefix);
+  const newerChannel = getNewerClusterVersionChannel(similarChannels, currentChannel);
+  const newerChannelVersion = splitClusterVersionChannel(newerChannel)?.version;
+  return isLoaded && !_.isEmpty(updateData) && isEditable
     ? [
         <NotificationEntry
-          actionPath="/settings/cluster"
+          actionPath="/settings/cluster?showVersions"
           actionText="Update cluster"
-          key="cluster-udpate"
+          key="cluster-update"
           description={updateData[0].version || 'Unknown'}
           type={NotificationTypes.update}
           title="Cluster update available"
           toggleNotificationDrawer={toggleNotificationDrawer}
-          targetPath="/settings/cluster"
+          targetPath="/settings/cluster?showVersions"
         />,
+        newerChannel && (
```

Comment:
Won't this only show the new channel notification if there an also an update available? These checks should be independent.

Reply:
Yeah, you're right.  Lemme fix.

Reply:
Done.